networkit/cpp/edgescores/test/EdgeScoreGTest.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+ * EdgeScoreGTest.cpp
+ *
+ *  Created on: 08.11.2025
+ *  Authors: Andreas Scharf (andreas.b.scharf@gmail.com)
+ */
+
+#include <gtest/gtest.h>
+
+#include <networkit/edgescores/EdgeScore.hpp>
+#include <networkit/graph/Graph.hpp>
+
+namespace NetworKit {
+
+class EdgeScoreTest : public ::testing::Test {
```

Comment:
The name of the class should be `EdgeScoreGTest` as referenced in the `CMakeLists.txt` file.